relay-general/src/protocol/stacktrace.rs

```diff
@@ -104,6 +107,7 @@ pub struct Frame {
     ///
     /// Setting this attribute to `false` causes the frame to be hidden/collapsed by default and
     /// mostly ignored during issue grouping.
+    #[metastructure(skip_serialization = "empty")]
```

Comment:
`Empty` for booleans and numbers is always false, so let's use `null` in those instances, too.